index.html

```diff
@@ -474,7 +474,74 @@ <h2 id="movie-title"></h2>
             releaseDate: '2016-05-27',
             releaseDateReadable: 'May 27, 2016',
             thumbnailUrl: 'https://upload.wikimedia.org/wikipedia/en/1/12/The_Do-Over_Poster.png',
-            ratings: []
+            ratings: [
+              {
+                name: 'IMDb',
+                value: '4.9/10',
+                link: 'http://www.imdb.com/title/tt4769836/'
+              },
+              {
+                name: 'Rotten Tomatoes',
+                value: '5%',
+                link: 'https://www.rottentomatoes.com/m/the_do_over_2016/#contentReviews'
+              },
+              {
+                name: 'Metacritic',
+                value: '22%',
+                link: 'http://www.metacritic.com/movie/the-do-over'
+              },
+              {
+                name: 'CinemaBlend',
+                value: '2.5/5',
+                link: 'https://www.cinemablend.com/reviews/Do-Over-70347.html'
+              }
+            ]
+          },
+          {
+            title: 'Sandy Wexler',
+            releaseDate: '2017-04-14',
+            releaseDateReadable: 'Apr 14, 2017',
+            thumbnailUrl: 'https://upload.wikimedia.org/wikipedia/en/9/92/Sandy_Wexler.jpg',
+            ratings: [
+              {
+                name: 'IMDb',
+                value: '5.1/10',
+                link: 'http://www.imdb.com/title/tt5893332/'
+              },
+              {
+                name: 'Rotten Tomatoes',
+                value: '32%',
+                link: 'https://www.rottentomatoes.com/m/sandy_wexler/#contentReviews'
+              },
+              {
+                name: 'Metacritic',
+                value: '22%',
+                link: 'http://www.metacritic.com/movie/sandy-wexler'
+              }
+            ]
+          },
+          {
+            title: 'The Meyerowitz Stories (New and Selected)',
```

Comment:
That's kinda against the idea, but I'll allow it.

Reply:
:1st_place_medal: